### PR TITLE
Fix script for Windows

### LINKED ISSUE #1390: [BUG] yarn build is not working on Windows
**What is the bug?**
When trying to run command "yarn build" command on Windows it shows: error Command failed with exit code 1,
rename build/securityDashboards-3.0.0.zip to build/security-dashboards-3.0.0.0.zip
'mv' is not recognized as an internal or external command.
Even when I tried running "yarn build --inline-builds --windows"

**How can one reproduce the bug?**
1. Go to \OpenSearch-Dashboards\plugins\security-dashboards-plugin
2. Run "yarn build" or "yarn build --inline-builds --windows" on terminal

**What is your host/environment?**
 - OS: Windows

**Do you have any screenshots?**

Screenshot 2023-03-30 230955
Screenshot 2023-03-30 231029

### PR DESCRIPTION
**Pull request by nurbq**
### Description
Fixes yarn build on Windows. #1390 
### Category
Bug fix.

### Why these changes are required?
To successfully build project on Windows.

### What is the old behavior before changes and new behavior after changes?
It was an error building the project on Windows machines. This changes brings successfully run on Windows machines also on Linux.

### Issues Resolved
#1390 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build_tools/rename_zip.js [hunk 1]
```diff
@@ -23,4 +23,13 @@ const oldName = `build/${osdJson.id}-${osdJson.opensearchDashboardsVersion}.zip`
 const newName = `build/${pluginName}-${packageJson.version}.zip`;
 
 console.log('rename ' + oldName + ' to ' + newName);
-shell(`mv ${oldName} ${newName}`);
+const os = require('os')
+
+if(os.type() == 'Windows_NT'){
```
Comment (peternied) [inline build_tools/rename_zip.js:28]: Can we remove the platform check and only use renameSync?

Comment (nurbq) [reply build_tools/rename_zip.js:28]: Then will it work on Linux?

Comment (peternied) [reply build_tools/rename_zip.js:28]: Yes

Comment (nurbq) [reply build_tools/rename_zip.js:28]: Ok then

Comment (nurbq) [reply build_tools/rename_zip.js]: I removed

**Review by peternied — APPROVED** [review]

**Review by scrawfor99 — DISMISSED** [review]
Great job!

**Review by RyanL1997 — APPROVED** [review]

**Review by DarshitChanpura — CHANGES_REQUESTED** [review]
Hey @nurbq, I think you meant to push changes with an if condition check for this script execution based on the underlying os:
```
const os = navigator.userAgent
if (os.search('Windows') !== -1) {
    const fs = require('fs');
    fs.renameSync(oldName, newName);
} else {
    const shell = require('child_process').execSync;
    shell(`mv ${oldName} ${newName}`);
}
```

**Review by DarshitChanpura — APPROVED** [review]

**Review by DarshitChanpura — COMMENTED** [review]
You can run `yarn lint --fix` to fix the linter error.

**Review by peternied — APPROVED** [review]

**Pull request merged**